src/config/reader.rs

```diff
@@ -116,20 +116,32 @@ impl ConfigReader {
                     }
                 }
                 LinkType::Protobuf => {
-                    let mut descriptors = self
-                        .resolve_descriptors(self.read_proto(&source.path).await?)
-                        .await?;
+                    let parent_descriptor = self.read_proto(&source.path).await?;
+                    let id = parent_descriptor
+                        .package // we should ideally take the package name as default id
+                        .clone()
+                        .unwrap_or(
```

Comment:
in case if package name is not defined in proto file to fetch the service with protox we need to pass just pure service_name without any package. But to link `@grpc` with specific `@link` we need some kind of id.
we need to add more tests for this with different cases of package naming here

Reply:
if we fix the `protox_parse::parse` issue (in comments below) then we can grantee some kind of id. Because in case of server reflection, we have 3 ways, but in all the ways it's expected that the proto file must have package name or file name.

but in our case we have freedom to load proto files over url.. we need to think of some wasy to give proper file name instead of path

Reply:
but I think that LinkType::Protobuf will always have `id` field. So we won't need to care about anything. But user might get confused because sometimes link id would be used or maybe package id or file name